android/src/main/java/com/brentvatne/common/API/SubtitleStyle.kt

```diff
@@ -1,6 +1,6 @@
 package com.brentvatne.common.API
 
-import com.brentvatne.ReactBridgeUtils
+import com.brentvatne.common.toolbox.ReactBridgeUtils
```

Comment:
look like you move the file, but not re add it ?

Reply:
Actually I just connect [ReactBridgeUtils.kt](https://github.com/react-native-video/react-native-video/blob/master/android/src/main/java/com/brentvatne/common/toolbox/ReactBridgeUtils.kt) and remove old file([ReactBridgeUtils.java](https://github.com/react-native-video/react-native-video/blob/master/android/src/main/java/com/brentvatne/ReactBridgeUtils.java))
these files are already existed

Reply:
Ok thank you, good catch then !